src/main/java/com/atlassian/db/replica/internal/aurora/AuroraJdbcUrl.java

```diff
@@ -1,12 +1,18 @@
 package com.atlassian.db.replica.internal.aurora;
 
-public class AuroraJdbcUrl {
-    private static final String PREFIX = "jdbc:postgresql://";
+import com.atlassian.db.replica.spi.jdbc.JdbcProtocol;
+import com.atlassian.db.replica.spi.jdbc.JdbcUrl;
 
+public class AuroraJdbcUrl extends JdbcUrl {
```

Comment:
It's internal. I guess we don't want to call it from the product codebase.

Reply:
URI gives us stricter validation of the input than the current implementation. Is it intentional?

Reply:
will use URI's validation when constructing JdbcUrl